### DISCUSSION TITLE
Could you please provide the parameters setting for SiamRPN++ in other benchmarks?

### DISCUSSION THREAD

**Discussion by jensenzhoujh** [STVIR/pysot#109] [General]
Could you plz provide the parameters setting for SiamRPN++ in UAV123, LaSOT, TrackingNet?

Many thanks for you time.

**Comment by lb1100** [comment]
LaSOT, TrackingNet's parameter setting and models are the same as [siamrpn_r50_l234_dwxcorr](https://github.com/STVIR/pysot/tree/master/experiments/siamrpn_r50_l234_dwxcorr). The model and setting for UAV123 is different. We will upload them recently.

**Comment by jensenzhoujh** [comment]
> LaSOT, TrackingNet's parameter setting and models are the same as [siamrpn_r50_l234_dwxcorr](https://github.com/STVIR/pysot/tree/master/experiments/siamrpn_r50_l234_dwxcorr). The model and setting for UAV123 is different. We will upload them recently.

Hi, could I know when will the upload be done?

Many thanks for your time.

**Comment by jensenzhoujh** [comment]
@lb1100  @StrangerZhang
